plugins/woocommerce/src/Enums/CatalogVisibility.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the catalog visibility values.
+ */
+class CatalogVisibility {
```

Comment:
Also, I belive we need to update https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Enums/README.md to mention this enum.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/54001/commits/5d78609cc6fa817dabfdc8f4136394b81c6e31ac